README.md

```diff
@@ -1,264 +1,340 @@
 ```
+
 where
 
-* **Actual_Username** is the actual username used for authentication (if a username/password is used for authentication).
-* **OS_Name** is the Operating System the application is running on.
-* **OS_Version** is the version number of the Operating System.
-* **Hardware_Platform** is the Hardware Platform the application is running on.
-* **MQTT_Library_name** is the MQTT Client library being used.
-* **MQTT_Library_version** is the version of the MQTT Client library being used.
+* *Actual_Username* is the actual username used for authentication (if a username/password is used for authentication).
+* *OS_Name* is the Operating System the application is running on.
+* *OS_Version* is the version number of the Operating System.
+* *Hardware_Platform* is the Hardware Platform the application is running on.
+* *MQTT_Library_name* is the MQTT Client library being used.
+* *MQTT_Library_version* is the version of the MQTT Client library being used.
 
-## Branches
+## Versioning
 
-### master branch
-The [master](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/master) branch hosts the continuous development of the AWS IoT Embedded C SDK (C-SDK) libraries. Please be aware that the development at the tip of the master branch is continuously in progress, and may have bugs. Consider using the [tagged releases](https://github.com/aws/aws-iot-device-sdk-embedded-C/releases) of the C-SDK for production ready software.
+C-SDK releases will now follow a date based versioning scheme with the format YYYYMM.NN, where:
 
-### v4_beta_deprecated branch (Formerly named v4_beta)
-The [v4_beta_deprecated](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v4_beta_deprecated) branch contains a beta version of the C-SDK libraries, which is now deprecated. This branch was earlier named as *v4_beta*, and was renamed to *v4_beta_deprecated*. The libraries in this branch will not be released. However, critical bugs will be fixed and tested. No new features will be added to this branch.
+* Y represents the year.
+* M represents the month.
+* N represents the release order within the designated month (00 being the first release).
 
+For example, a second release in June 2021 would be 202106.01. Although the SDK releases have moved to date-based versioning, each library within the SDK will still retain semantic versioning. In semantic versioning, the version number itself (X.Y.Z) indicates whether the release is a major, minor, or point release. You can use the semantic version of a library to assess the scope and impact of a new release on your application.
 
 ## Releases
 
 All of the released versions of the C-SDK libraries are available as git tags. For example, the last release of the v3 SDK version is available at [tag 3.0.1](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v3.0.1).
 
-## Versioning
+### 202011.00
```

Comment:
@abhidixi11  pls confirm what and when this section should be filled, and how this PR should be handled.